modules/plottinggl/glsl/legend.frag

```diff
@@ -75,8 +75,8 @@ void main() {
     // blend in the checkerboard as background to the TF depending on its opacity
     vec4 finalColor = over(checkerBoard(centeredPos), colorTF);
 
-    // set border flag iff the fragment coord is within the border
-    bool border = borderWidth > 0 && any(greaterThan(abs(centeredPos), outputDim * 0.5));
+    // set border flag if the fragment coord is within the border
+    float border = borderWidth > 0 && any(greaterThan(abs(centeredPos), outputDim * 0.5)) ? 1.f : 0.f;
```

Comment:
You should not use `f` as float specifier in GLSL shaders!

any particular reason for using float and not int? Could you test
```suggestion
    int border = borderWidth > 0 && any(greaterThan(abs(centeredPos), outputDim * 0.5)) ? 1 : 0;
```

Reply:
This is the wrong fix... it should be
FragData0 =  mix(finalColor, color, bvec4(border));

Reply:
> You should not use `f` as float specifier in GLSL shaders!

Okay, then I'll remove the ones six lines above too :)

Reply:
> This is the wrong fix... it should be
> FragData0 = mix(finalColor, color, bvec4(border));

Seem to work too